### DISCUSSION TITLE
Explain me about meeting room management.

### DISCUSSION THREAD

**Discussion by ShineThyuZan** [community/community#62566] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Someone can join for descussion

**Comment by Scholar17** [comment]
I am working for Android Development, Testing, Documentation, Deployment and Operation. Occasionally, I take the responsibility for managing the meeting room. I want to tell you about how i manage the meeting room. In our company, we have three meeting rooms. Firstly, I booked a meeting room reservation system at our company. With this system, I can view vacant meeting rooms on schedule. In order to make a reservation, I need to fill in a meeting reservation form which includes username, meeting room number, meeting start time, meeting end time and meeting name. After that, I submit the form and the meeting room booking is finished. Then, I announced the meeting information via slack application and email. If we need to contact a person who is away from our company, we use the zoom meeting platform, but sometimes we use Google Meet and Microsoft team. We use the Cisco brand for video devices and the Samsung brand for monitors. In our knowledge sharing session, we use a projector device which is view sonic brand. Before the meeting, we ensure to complete necessary steps. We went to the meeting room 15 minutes ago before the meeting started and prepared for the meeting, such as checking network connection, checking required devices such as laptop, camera, monitor, air conditioner and projector. After that, we start meeting via a zoom meeting platform or onsite.

**Comment by ShineThyuZan** [comment]
Noted , Can I contact you?
Ok in google meet?

**Comment by samus-aran** [comment]
@ShineThyuZan 

Thanks for being a part of the GitHub Community, we're glad you're here!

In a number of your past discussions it appears you're looking for help for very generalised advice on using software, yet show technical knowledge in others. You might want to try asking for help somewhere that focuses on how particular software works and how to implement it into a company elsewhere. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. 
We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
